schema/descriptor.schema.json

```diff
@@ -257,25 +296,50 @@
                         "id": "http://github.com/boutiques/boutiques-schema/input/max-list-entries",
                         "description": "Specify the maximum number of entries in the list. May only be used with List type inputs.",
                         "type": "number"
+                    },
+                    "absolute-path": {
+                        "id": "http://github.com/boutiques/boutiques-schema/input/absolute-path",
+                        "description": "Specifies that this input must be given as an absolute path. Only specifiable for File type inputs. Cannot be true if relative-path is true.",
+                        "type": "boolean"
+                    },
+                    "relative-path": {
+                        "id": "http://github.com/boutiques/boutiques-schema/input/relative-path",
```

Comment:
The idea was that if none of these properties are here, then it is up to the implementation to decide which one to use. There might be good reasons to use one or the other. In CBRAIN for instance, relative paths are always preferred because it makes the tasks relocatable in different clusters (which is not really an issue with containers but still is the preferred way). 

But I think I'm fine with a single `use-absolute-path` boolean property, the default being to use relative paths.